init/Kconfig

```diff
@@ -1156,6 +1156,12 @@ config HAVE_PCSPKR_PLATFORM
 config BPF
 	bool
 
+config LUNATIK
+	bool "Lunatik"
+	default n
+	help
+	  Support for the Lua interpreter
```

Comment:
really cool =).. but it should go on lunatik repo, right?

Reply:
Fair enough